wiki/Troubleshooting/osu!stable/en.md

```diff
@@ -0,0 +1 @@
+# osu!stable troubleshooting
```

Comment:
?

Reply:
I thought it's necessary to prevent not found pages.